quantum/util/impl/quantum_spinlock_util_impl.h

```diff
@@ -216,4 +421,24 @@ size_t& SpinLockUtil::numSpins()
     return _numSpins;
 }
 
+inline constexpr
+uint32_t SpinLockUtil::set(int16_t h, int16_t l) {
+    return ((uint32_t)h<<16) | (l&mask);
+}
+
+inline constexpr
+int16_t SpinLockUtil::getHigh(uint32_t n) {
```

Comment:
While this is accurate, I'm tempted to say we should rename getHigh & getLow to something more indicative of the values stored in the high & low halves of the int; it's very easy when reading this code to forget which value is stored in which half.

Reply:
Ok renamed to upgrades() and owners() respectively.